### DISCUSSION TITLE
Lock sounds

### DISCUSSION THREAD

**Discussion by marcone** [marcone/teslausb#808] [General]
has anybody poked around the limits of lock sounds to determine what's possible and what isn't? I know the official statement is that a lock sound must be called LockChime.wav and be < 1MB, but:
\- is 1MB really the limit?
\- does it need to be camelcase "LockChime.wav", or do names like "lockchime.wav" and "LockChime.WAV" also work?
\- will an mp3 that's renamed to .wav work?
\- if it must be an actual .wav file, what encodings are supported (e.g. is mp3-in-wav supported, or must it be PCM?)
\- if PCM, what bit depths and sampling rates are supported?
(I don't have a PWS otherwise I would have checked myself)

**Comment by Arthur-Philip-Dent** [comment]
Don’t have a PWS either (2019 M3P). 
Could post your question in our local Tesla club (TFF, Germany) for help? 

I would do anything to get a retrofit boombox for my car and activation in the software… (is that true, that even with the right hardware it wouldn’t do without entry in the central repository?)

**Comment by iFredLouzada** [comment]
I just started using teslausb. I can tell you all mp3 files that i renamed to .wav worked, but dont know if its coincidence. I wanted to use teslausb only for boombox and the music folder for the lockchime, but I have not been able to find a way to disable the cam.
I already have a 1tb ssd on the glovebox and wanted to use teslausb only on the main console port for again, boombox and lockchime.
